lib/run.rb

```diff
@@ -8,10 +8,17 @@
 require "bot"
 require "configuration/jira"
 
+def get_array_from_env(key)
+  JSON.parse(ENV[key])
+rescue TypeError
```

Comment:
I guessed we would expect a TypeError if the environment variable for some reason isn't set. As it would then be trying to parse nil, and the console gives me
```ruby
[1] pry(main)> JSON.parse nil
TypeError: Nil is not a valid JSON source.
```
for this. Is this reasoning correct? Or could there be other errors?
What about a JSON::ParserError if the value was somehow inprocessable, do we need to cover this?

Reply:
I guess it doesn't hurt to just rescue both of these